license/ilbc-freeware/index.md

```diff
@@ -10,8 +10,8 @@ crumb: iLBC Freeware
 
 
 
-Global IP Solutions, the creator of iLBC, was acquired by Google in January
-2011. Google continues to support the iLBC effort and is offering it as part
+Global IP Solutions, the creator of iLBC, was acquired by Google in
+January 2011. Google continues to support the iLBC effort and is offering it as part
```

Comment:
Without this, the `2011.` gets rendered as a list:
![image](https://user-images.githubusercontent.com/1538624/102585592-a5933880-40d6-11eb-9aed-91f8c95d1a63.png)